app/avatar/models.py

```diff
@@ -64,6 +64,10 @@ class Meta:
     )
 
     use_github_avatar = models.BooleanField(default=True)
+    recommended_by_staff = models.BooleanField(default=False)
+    active = models.BooleanField(default=False)
+    profile = models.ForeignKey('dashboard.Profile', on_delete=models.CASCADE, null=True, blank=True,
```

Comment:
Migrating to this flow and `active` versus `github` or `custom` changes quite a bit fundamentally of how `Avatars` are supposed to work...  If we're marking one as active, shouldn't we deprecate the use_github_avatar and platform specific fields, since we're allowing them to create `n` number of `Avatar` objects to be associated with their `Profile` and simply default to the GH avatar to start? cc @owocki

Reply:
@mbeacom take a look at this: https://github.com/kziemianek/web/commit/37799d49ad2ecd254dc673bb00c704687c8711c7

does this aproach makes more sense to You ?
I like it more, because `Profile` still contains active `Avatar` and `CustomAvatar` is a snapshot which can be restored by the user.

`recommended_by_staff` and `config` can also be moved to `CustomAvatar`.

Ping me back if we want to move into that direction.